common/common.csproj

```diff
@@ -1,185 +1,187 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
wtf is happening here?

Reply:
No idea, in my local commit history this only shows the addition of line 91+92 as committed changes, no idea why or how it replaced the entire file on pushing.